pkgs/development/libraries/libxml2/default.nix

```diff
@@ -32,7 +32,7 @@ stdenv.mkDerivation rec {
   outputs = [ "bin" "dev" "out" "man" "doc" ]
     ++ lib.optional pythonSupport "py"
     ++ lib.optional (enableStatic && enableShared) "static";
-  propagatedBuildOutputs = [ "out" "bin" ] ++ lib.optional pythonSupport "py";
+  propagatedBuildOutputs = [ "out" "bin" ];
```

Comment:
If the [docs](https://nixos.org/nixpkgs/manual/#sec-multiple-outputs-using-split-packages) are correct, we should not need this line at all:

> `propagatedBuildOutputs` of that package which by default contain `$outputBin` and `$outputLib`

Reply:
Looks like that is still he case: https://github.com/NixOS/nixpkgs/blob/a8d1f3233bbd0862a643cc4806f8f9cc4edcbc04/pkgs/build-support/setup-hooks/multiple-outputs.sh#L182

Reply:
i removed the whole line on master and was able to build a few hundred packages fine :)